src/spikeinterface/widgets/unit_waveforms.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         set_title=True,
         same_axis=False,
         shade_templates=True,
-        templates_percentile_shading=[1, 25, 75, 98],
+        templates_percentile_shading=[1, 25, 75, 99],
```

Comment:
Wouldn't it be better to have this as a tuple then for the default. The end-user can give a list, but we have the default be immutable.

Reply:
done